cmd/jcsd/main.go

```
@@ -0,0 +1,79 @@
+	}
+
+	if req.Op != "canon" {
+		fmt.Fprintln(os.Stderr, "Unsupported op:", req.Op)
+		encoder.Encode(Response{Error: "unsupported op"})
+		return
+	}
+
+	canon, err := jsoncanonicalizer.Transform([]byte(req.Source))
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Canonicalization error:", err)
+		encoder.Encode(Response{Error: err.Error()})
```

golangci-lint (1.64.8)

Error return value of `encoder.Encode` is not checked

(errcheck)